World.py

```diff
@@ -78,7 +78,16 @@ def __init__(self, world_id: int, settings: Settings, resolve_randomized_setting
         self.disable_trade_revert: bool = self.shuffle_interior_entrances or settings.shuffle_overworld_entrances or settings.adult_trade_shuffle
         self.skip_child_zelda: bool = 'Zeldas Letter' not in settings.shuffle_child_trade and \
                                       'Zeldas Letter' in self.distribution.starting_items
-        self.selected_adult_trade_item: str = ''
+        self.selected_adult_trade_item: str = random.choice(settings.adult_trade_start) if settings.adult_trade_start else None
```

Comment:
This should check both `settings.adult_trade_start` and `not settings.adult_trade_shuffle`. I'm not sure it hurts anything as is, but it's quicker to add the condition then it is for me to verify full trade shuffle is unaffected (both logic and the rom flag).